### ISSUE TITLE
Nearly 3 points below the provided VQAv2 fine-tuning result

### ISSUE THREAD

**Issue by 4444xhc** [dandelin/ViLT#19]
Hi,

Thank you for releasing the code!

I want to have a try on getting the result of fine-tuning on VQAv2 recently.
The result generated by the provided vilt_vqa.ckpt (ViLT-B/32 200k finetuned on VQAv2) is [{"test-dev": {"yes/no": 87.44, "number": 50.2, "other": 62.38, "overall": 71.32}}]. 

Then, I used "num_gpus=4 num_nodes=1 task_finetune_vqa_randaug per_gpu_batchsize=16 load_path=".../vilt_200k_mlm_itm.ckpt"" to fine-tune on VQAv2 using my GPUs and uploaded to VQA Challenge 2021 to evaluate. The result was [{"test-dev": {"yes/no": 85.28, "number": 46.63, "other": 58.87, "overall": 68.36}}]. 

Will there be the same result if I use the same settings?

**Comment by dandelin** [comment]
I didn't test gradient accumulation for max_epoch based training (=fine-tuning), unlike max_step setting (=pre-training) I've set max_step manually in the max_epoch setting.

Going over the code again, I suspect this line  https://github.com/dandelin/ViLT/blob/master/vilt/modules/vilt_utils.py#L247 could be the source of the degradation.

If I'm guessing it right, the gradient accumulation in current fine-tuning will follow wrong, condensed learning rate schedule.

Please delete the line and try again.
+ Can you report lr summary in the tensorboard log?

**Comment by 4444xhc** [comment]
@dandelin Thank you for your advice and time. I happened to find that I made some mistakes during the dataset preparation. The code and paper provide sufficient information. Now, I can achieve the same VQAv2 test-dev result as the paper. 
Thank you again for your patient help. I will close this issue.
